src/cpp/session/modules/SessionProfiler.R

```diff
@@ -80,3 +101,11 @@
       .rs.enqueClientEvent("rprof_stopped");
    }
 })
+
+options("profvis.print" = function(x)
```

Comment:
Yes, that's how we handle several other hook-style options. e.g. https://github.com/rstudio/rstudio/blob/master/src/cpp/r/R/Options.R#L27-L42